### ISSUE TITLE
Crash after the mess up screen shown.

### ISSUE THREAD

**Issue by Dunard** [GermanAizek/octavius-game#1]
mitchell@nobara-pc:~/octavius-game/res$ ./octavius
You now have 1000 credits.
Segmentation fault (core dumped)
![Screenshot_20240417_103506](https://github.com/GermanAizek/octavius-game/assets/31096423/eefeb6c4-0485-48f1-a5f9-eb80acee4f3c)

Using Nobara 39 (Fedora 39 clone.)

mitchell@nobara-pc:~/octavius-game/build$ cmake .. -G "Ninja"
-- The C compiler identification is GNU 13.2.1
-- The CXX compiler identification is GNU 13.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found OpenGL: /usr/lib64/libOpenGL.so  found components: OpenGL 
-- Found GLUT: /usr/lib64/libglut.so  
-- Configuring done (0.5s)
-- Generating done (0.0s)
-- Build files have been written to: /home/mitchell/octavius-game/build
mitchell@nobara-pc:~/octavius-game/build$ ninja
[7/7] Linking CXX executable octavius
mitchell@nobara-pc:~/octavius-game/build$ cd /home/mitchell/octavius-game/res/

Paste the octavius over into the /home/mitchell/octavius-game/res/

./octavius

**Comment by GermanAizek** [comment]
@Dunard, to be honest, this is very old code, I have never done development and testing for fedora. This linux distribution should not be very different from others.

**Comment by Dunard** [comment]
I got it to work on Ubuntu 22.04 (Distrobox version). The skins too far apart. I tried on the VM and I almost got the skins into the right place.
![Screenshot_20240418_110233](https://github.com/GermanAizek/octavius-game/assets/31096423/c366e36a-4402-4ce2-9234-bddbb1d2fb5c)

Screenshot from the Ubuntu 22.04 (Distrobox version) screen. Just three issues it having. Don't worry about it. I just tested the game out. :)
